overlay/mobile-nixos/kernel-append-dtbs/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, runCommand }:
```

Comment:
Cross @telent's appended DTB solution with this one.

 - https://github.com/NixOS/mobile-nixos/pull/506/commits/b95a8e80869961b1b737273764db3f956603f163
 - https://github.com/samueldr-wip/mobile-nixos-wip/commit/7e569af5d09d7dd183123abd5285ad8a2330f830

So the main change to do is to drop this builder, and do it in-line within the bootimg builder. The leftover question is how to properly expose this to the config.